examples/nucleo_g474re/servo_pwm/main.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2021, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <modm/board.hpp>
+
+using namespace Board;
+
+int
+main()
+{
+	Board::initialize();
+	LedD13::setOutput();
+
+	MODM_LOG_INFO << "modm example: Serve PWM" << modm::endl << modm::endl;
+	MODM_LOG_INFO << "Generates a 50 Hz PWM signal with a pulse width between 1 ms and 2 ms suitable" << modm::endl;
+	MODM_LOG_INFO << "for many servo motors or RC electronic speed controllers on pin B7." << modm::endl;
+	MODM_LOG_INFO << "1.5 ms pulse width is output when the user button is pressed, " << modm::endl;
+	MODM_LOG_INFO << "otherwise 1 ms pulse width." << modm::endl << modm::endl;
+
+	Timer4::connect<GpioB7::Ch2>();
+	Timer4::enable();
+	Timer4::setMode(Timer4::Mode::UpCounter);
+
+	constexpr uint16_t overflow = 0xffff;
+	constexpr uint16_t minPwm = (overflow / 20) * 1.0; // 1 ms pulse width
+	constexpr uint16_t targetPwm = (overflow / 20) * 1.5; // 1.5 ms pulse width
+	constexpr uint16_t maxPwm = (overflow / 20) * 2.0; // 2 ms pulse width
```

Comment:
The division is done in integer arithmetic but the multiplication is float. Did you want to do that?

Reply:
Nop, thanks for pointing that out...